### PR TITLE
Improve activation file example

### PR DESCRIPTION
**Pull request by goralczm**
Original example structure was not showing in Action tab.

#### Changes

- Added unityVersion to activation file request
- Update upload-artifact version to latest

#### Checklist

- [x] Read the contribution [guide](../blob/main/CONTRIBUTING.md) and accept the
      [code](../blob/main/CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: docs/03-github/02-activation.mdx [hunk 1]
```diff
@@ -37,20 +37,24 @@ Create a file called `.github/workflows/activation.yml` and add the below workfl
 
 ```yaml
 name: Acquire activation file
+
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
```
Comment (webbertakken) [inline docs/03-github/02-activation.mdx:42]: the object braces were there to make it less ambiguous for people that the key has no explicit content.

File: docs/03-github/02-activation.mdx [hunk 2]
```diff
@@ -37,20 +37,24 @@ Create a file called `.github/workflows/activation.yml` and add the below workfl
...
     runs-on: ubuntu-latest
     steps:
-      # Request manual activation file
       - name: Request manual activation file
         id: getManualLicenseFile
         uses: game-ci/unity-request-activation-file@v2
-      # Upload artifact (Unity_v20XX.X.XXXX.alf)
+        with:
+          unityVersion: 202x.x.xfx
+          # Include your unity version
```
Comment (webbertakken) [inline docs/03-github/02-activation.mdx:54]: Did you find a use-case where you needed the license to be for a specific version? As far as I know this should work with the default (which is `'2019.2.11f1'`) for all versions.

If not; do you still think we should make this change? i.e. will it make it more understandable for others. If yes, we'll merge it.

**Review by webbertakken — APPROVED** [review]

File: docs/03-github/02-activation.mdx (hunk 2 above)
Comment (goralczm) [reply docs/03-github/02-activation.mdx:54]: I will check a few thing tomorrow to not force something useless but I had a big struggle with the original example as it was not showing in my Actions in repo.
I tried a lot of thing and finally managed to show it with this structure.

Comment (goralczm) [reply docs/03-github/02-activation.mdx:54]: Let me make sure and I will get back on that

Comment (goralczm) [reply docs/03-github/02-activation.mdx:54]: So it was just my mistake as I just started with github action.
So I will close it then and btw great job this amazing project!

Comment (webbertakken) [reply docs/03-github/02-activation.mdx:54]: Thanks for double checking and for the intention to help us improve GameCI :)

**Pull request closed without merging**